modules/l0/src/main/scala/org/elpaca_metagraph/l0/rewards/ElpacaRewards.scala

```diff
@@ -68,6 +68,29 @@ object ElpacaRewards {
             dataSource.addressesToReward.collect {
               case (address, ds) if ds.epochProgressToReward.value == currentEpochProgress.value => address -> ds.amountToReward
             }
+
+          case dataSource: InflowTransactionsDataSource =>
+            dataSource.existingWallets.foldLeft(Map.empty[Address, Long]) { (acc, dsInfo) =>
+              val (_, ds) = dsInfo
+              ds.addressesToReward.collect {
+                case inflowAddressInfo if inflowAddressInfo.epochProgressToReward.value == currentEpochProgress.value =>
+                  inflowAddressInfo.addressToReward -> inflowAddressInfo.amountToReward
+              }.foldLeft(acc) { case (innerAcc, (addressToReward, rewardAmount)) =>
+                innerAcc.updated(addressToReward, innerAcc.getOrElse(addressToReward, 0L) + rewardAmount)
```

Comment:
Instead of 0L it should be Amount.empty out sth like that. Try to avoid magic numbers in the code